### ISSUE TITLE
🚀 [REQUEST] - add decoding="async" together loading="lazy"

### ISSUE THREAD

**Issue by maofree** [Oksydan/falcon#377]
### Summary

Hi @Oksydan 
right now I've found this solution to increase the loading page speed
add decoding="async" with loading="lazy"
look at here
https://fantinel.dev/web-images-modern-formats#lazy-loading-and-async-decoding

what do you think?

I've seen that astro's website use it in its source code
https://astro.build/

I'm going to test it

bye

### Feature target required module/theme

theme

### Drawbacks

increase the page speed

**Comment by maofree** [comment]
It's not easy to understand, but I think it earns some points

**Comment by Oksydan** [comment]
Hi @maofree,

yes, using `decoding="async"` isn't a bad idea. 
I am adding this to Falcon 4 roadmap.

**Comment by StefanVonRanda** [comment]
Agreed, `decoding="async"` is a good default, but `loading="lazy"` should not be used on any image that visible on load.
You could implement some sort of bounding box check to detect images not currently in view on load, but that defeat the purpose.
